clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/cache/Keys.groovy

```diff
@@ -69,7 +69,7 @@ class Keys {
         break
       case Namespace.LOAD_BALANCERS.ns:
         def names = Names.parseName(parts[4])
-        result << [application: names.app, account: parts[2], name: parts[4], namespace: parts[3], stack: names.stack, detail: names.detail]
+        result << [application: names.app, account: parts[2], name: parts[4], loadBalancer: parts[4], namespace: parts[3], stack: names.stack, detail: names.detail]
```

Comment:
Needed since the `urlMappingTemplate` for `LOAD_BALANCERS` in `AwsProvider` refers to `$loadBalancer`.